plugins/module_utils/network/ios/config/bgp_address_family/bgp_address_family.py

```diff
@@ -28,11 +28,11 @@
     Facts,
 )
 from ansible_collections.cisco.ios.plugins.module_utils.network.ios.rm_templates.bgp_address_family import (
-    Bgp_AddressFamilyTemplate,
+    Bgp_address_familyTemplate,
```

Comment:
Not sure why we need to change these. These are generated by the cli_rm_builder `{{ resource|capitalize }}`.

Reply:
The module name is produced consistent with cli_rm_builder's naming pattern. What it occurs to me during scaffolding the RM builder was not used properly then. Addressed all the Class name changes in the PR.